apps/fluent-tester/src/FluentTester/TestComponents/Button/ButtonHOCTestSection.tsx

```diff
@@ -38,7 +38,8 @@ export const ButtonHOCTest: React.FunctionComponent = () => {
       {svgIconsEnabled && <CustomIconButton icon={{ svgSource: svgProps }}>Customized Icon Button</CustomIconButton>}
       <Button
         style={commonTestStyles.vmargin}
-        onClick={() => {
+        onClick={(e) => {
+          console.log(e.timeStamp);
```

Comment:
This _might_ be a windows only prop. I can try testing this on macOS and see what happens.

Reply:
Ugh it's probably timestamp on other platforms and timeStamp on win32 or something like that. /shakesfist

Reply:
Oh it's timeStamp on the outer event and timestamp on the inner native event.